src/components/bottom/daily-digest/main.js

```diff
@@ -0,0 +1,32 @@
+import myft from 'next-myft-client';
+
+module.exports = function customSetup (banner, done) {
+
+	if (window.FT && window.FT.flags && window.FT.flags.oneClickDailyDigest) {
+
+		function handleSignUpClick (evt) {
+			evt.preventDefault();
+			const conceptId = document.documentElement.dataset.conceptId;
+			if (conceptId) {
+				function addUserToConcept () {
+					return myft.add('user', null, 'followed', 'concept', conceptId);
+				}
+				function logError (err) {
+					/* eslint no-console:0 */
+					console.log({ info: 'could not add user to cocnept', conceptId, err });
```

Comment:
I believe there is a typo here:
```suggestion
					console.log({ info: 'could not add user to concept', conceptId, err });
```